main.cpp

```diff
@@ -67,6 +68,25 @@ int main(int argc, char* argv[]) {
   std::shared_ptr<Texture> texture = std::make_shared<Texture>("data/font/main");
 
   std::shared_ptr<Text> text = std::make_shared<Text>("data/font/liberation_sans_bold_65");
+  auto drawText = [&](glm::vec2 position, std::string message) {
```

Comment:
Yes, but it's beyond the scope of this PR. I'm still not sure what actual features the text renderer will require.
So instead of adding complexity now (which may be unnecessary / removed later), we can just keep this hacky temporarily and then decide what we really need, once and for all (when we have more logic implemented).